router/server.go

```diff
@@ -187,21 +187,22 @@ func (s *Server) isLegalRequest(req *http.Request) bool {
 	return true
 }
 
-func (s *Server) resendBucky(orgReq *http.Request) {
-	buf, _ := ioutil.ReadAll(orgReq.Body)
-	newBody := ioutil.NopCloser(bytes.NewBuffer(buf))
-	orgBody := ioutil.NopCloser(bytes.NewBuffer(buf))
-
-	req, err := http.NewRequest("POST", "http://monitor.imqs.co.za/bucky/v1/send", newBody)
-	orgReq.Body = orgBody
+func (s *Server) resendBucky(w http.ResponseWriter, orgReq *http.Request) {
+	buf, err := ioutil.ReadAll(orgReq.Body)
+	if err != nil {
+		return
```

Comment:
We don't care if something goes wrong with this. It just sends the metric data through to the server. The hole idea was that the monitor stuff does not interfere with any thing else, and this is the best way to achieve that.